pkg/processor/webhook/mutating.go

```diff
@@ -64,7 +64,8 @@ func (w mwh) Process(appMeta helmify.AppMetadata, obj *unstructured.Unstructured
 	if err != nil {
 		return true, nil, errors.Wrap(err, "unable get webhook certName")
 	}
-	certName = strings.TrimPrefix(certName, appMeta.Namespace()+"/"+appMeta.ChartName()+"-")
```

Comment:
Thank you for reporting and fixing the issue.
Please alos update project samples by running:
```console
cat test_data/k8s-operator-kustomize.output | go run ./cmd/helmify examples/operator
```
and pushing updated samples into your branch.